src/core/client/admin/components/UserStatus/BanModal.tsx

```diff
@@ -264,15 +254,16 @@ const BanModal: FunctionComponent<Props> = ({
                     </Flex>
                   )}
 
-                  {!!moderationScopesEnabled &&
-                    updateType === UpdateType.SPECIFIC_SITES && (
-                      <UserStatusSitesList
-                        userBanStatus={userBanStatus}
-                        viewerScopes={viewerScopes}
-                        banState={[banSiteIDs, setBanSiteIDs]}
-                        unbanState={[unbanSiteIDs, setUnbanSiteIDs]}
-                      />
-                    )}
+                  {(viewerIsSingleSiteMod ||
```

Comment:
I think that the submit button disabled check also needs to be updated for the form now, so that it can't be submitted without the single site mod's site being checked. I think something like this will work:
```
disabled={
  ((updateType === UpdateType.ALL_SITES &&
  viewerIsSingleSiteMod) ||
  updateType === UpdateType.SPECIFIC_SITES) &&
  !pendingSiteBanUpdates
}
```